### PR TITLE
Reblog Display bugfixes

### LINKED ISSUE #641: Reblog Display Options Acting Odd
This screenshot was taken with the Style set to Flat, and the default options set.
new bitmap image
I think this is what Nested style is supposed to look like, based on the way the post described it.

This screenshot was taken of the same post, after Style was set to Nested, with no other options.
new bitmap image
Everything that would be underneath a blockquote reblog chain disappears.

Upon hitting the reblog button, everything in the blank post appears, as normal.

For whatever amount it helps, this is using Google Chrome on an HP laptop, and I made certain that Editable Reblogs was deactivated.

### PR DESCRIPTION
**Pull request by nightpool**
Fixes #641 #630

(possibly #644 but haven't tested it)

### CODE REVIEW

File: Extensions/better_reblogs.js [hunk 1]
```diff
@@ -248,8 +256,24 @@ XKit.extensions.better_reblogs = new Object({
...
+            var $el = $(this);
+            var val = $el.val();
+            $el.blur();
+            XKit.window.show("Warning", "Changing the reblog style requires refreshing the page. "+
+                "<br>Are you sure you wish to continue?",
+                "warning",
+                '<div id="xkit-confirm-refresh" class="xkit-button default">Refresh</div>'+
+                '<div id="xkit-close-message" class="xkit-button">Cancel</div>');
+            $("#xkit-confirm-refresh").click(function(){
+                window.location = location;
```
Comment (hobinjk) [inline Extensions/better_reblogs.js]: It looks like `location` is undefined here

Comment (nightpool) [inline Extensions/better_reblogs.js]: :+1:

**Comment by hobinjk** [comment]
Does not fix #644, otherwise looks good.
@homu r+

**Comment by nightpool** [comment]
@hobinjk were you able to reproduce #664? I haven't been able to.

File: Extensions/better_reblogs.js [hunk 2]
```diff
@@ -175,7 +179,7 @@ XKit.extensions.better_reblogs = new Object({
                     return;
                 }
                 title.remove();
-                parent.prepend(title);
+                parent.prepend();
```
Comment (nightpool) [inline Extensions/better_reblogs.js]: why didn't I notice this.
why did we not catch this during code review.

Comment (hobinjk) [inline Extensions/better_reblogs.js]: Oh wow I totally skimmed over this as
```
+                parent.prepend(title);
-                parent.prepend();
```

**Pull request merged**